content/download/index.html.haml

```diff
@@ -163,6 +163,10 @@ title: Jenkins installation and setup
                 %span.icon
                 %span.title
                   Ubuntu/Debian
+              %a.list-group-item.list-group-item-action{:href=>'http://pkg.openindiana.org/hipster/en/search.shtml%3Ftoken=jenkins&amp;action=Search'}
```

Comment:
I double-checked... via github-webgui that created the patch - it claims there are spaces. Maybe the "Improve this page" link is not as useful as it might be, though not Jenkins's fault ;)